flow/metro.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This file will get pulled in internally because `react-native-github/flow` is a lib dir for our xplat `.flowconfig` at facebook, right? Will this cause a problem?

Reply:
Might want to put this in like, an "oss-flow" folder @hramos.

Reply:
Ah, yes, this folder is used in the xplat `.flowconfig`.

Reply:
Moved to `flow-github`, to follow the convention established by the `react-native-github` folder in `xplat`.